### ISSUE TITLE
There's no way to upload and remove files from AWS S3.

### ISSUE THREAD

**Issue by vihuvac** [arvindr21/blueimp-file-upload-expressjs#51]
I've followed the instructions in order to make it work but I could not do it, in order to upload files into **S3** I had to edit the ```index.js``` file as follows:

```
...
var allFilesProccessed = false;

files.forEach(function(file, idx) {
    allFilesProccessed = true;
    file.proccessed = true;
});
...
```

The **processed** option was not being updated, so the **post** method died as pending.

I could not get to remove any file, the **request** to delete the file or files is being doing to the local host instead of **aws s3**.

Did someone have the same problem?

**Comment by Hiswe** [comment]
@vihuvac I had the same problem for the upload.

Using the 0.4.5 by making an `npm install arvindr21/blueimp-file-upload-expressjs` has resolved the problem for me.
So it should be good if the latest version were on npm :)

For the delete, I haven't checked yet.

**Comment by vihuvac** [comment]
@Hiswe yeah you are right, I had noticed it, passing ```arvindr21/blueimp-file-upload-expressjs``` through the ```package.json``` file, it worked! But it looks like the repo has not been updated in ```npm```.

The crazy thing! I saw that the images are being deleting but in the console there is an error alert like they were not being deleted, then the nodejs process is closed automatically due to this error. I couldn't get to solve it yet.

Thank you very much for your observation. :+1:

**Comment by ldarren** [comment]
i had similar problem, and i have it fixed in my fork https://github.com/ldarren/blueimp-file-upload-expressjs
